2022-07-22 (#161)
rustacean-station/rustacean-station.org: _episodes/interviews/2022-07-22-daniel-thompson.md

---
title: "Tauri with Daniel Thompson-Yvetot"

> date: 2022-07-29T17:30:00Z

Reviewer: ```suggestion
date: 2022-07-24T17:00:00Z
```

file: https://audio.rustacean-station.org/file/rustacean-station/2022-07-22-daniel-thompson.mp3
duration: "44:44"
length: "42945723"
#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
---

> Allen Wyma talks with [Daniel Thompson-Yvetot](https://github.com/nothingismagick), Co-founder at [Tauri](https://tauri.studio/). Tauri is a toolkit that helps developers make applications for the major desktop platforms - using virtually any frontend framework in existence.

Reviewer: ```suggestion
Allen Wyma talks with [Daniel Thompson-Yvetot](https://github.com/nothingismagick), co-founder of [Tauri](https://tauri.studio/). Tauri is a toolkit that helps developers make applications for the major desktop platforms using a variety of front-end frameworks.
```

Reviewer: The editing at the end there is just because we're not a marketing channel — let's focus on things that are actually true.
